docs/tutorials/index.md

```diff
@@ -99,96 +91,173 @@ $ ./bin/start-micro-quickstart
+   appears as expected. 
+
+   ![Data loader sample](../assets/tutorial-batch-data-loader-02.png "Data loader sample")
+
+   Notice that your position in the sequence of steps to load data, **Connect** in our case, appears at the top of the console, as shown below. 
+   You can click other steps to move forward or backward in the sequence at any time.
+   
+   ![Load data](../assets/tutorial-batch-data-loader-12.png)  
+   
+
+5. Click **Next: Parse data**. 
+
+   The data loader tries to determine the parser appropriate for the data format automatically. In this case 
+   it identifies the data format as `json`, as shown in the **Input format** field at the bottom right.
+
+   ![Data loader parse data](../assets/tutorial-batch-data-loader-03.png "Data loader parse data")
+
+   Feel free to select other **Input format** options to get a sense of their configuration settings 
+   and how Druid parses other types of data.  
+
+6. With the JSON parser selected, click **Next: Parse time**. The **Parse time** settings are where you view and adjust the 
+   primary timestamp column for the data.
+
+   ![Data loader parse time](../assets/tutorial-batch-data-loader-04.png "Data loader parse time")
+
+   Druid requires data to have a primary timestamp column (internally stored in a column called `__time`).
+   If you do not have a timestamp in your data, select `Constant value`. In our example, the data loader 
+   determines that the `time` column is the only candidate that can be used as the primary time column.
+
+7. Click **Next: Transform**, **Next: Filter**, and then **Next: Configure schema**, skipping a few steps.
+
+   You do not need to adjust transformation or filtering settings, as applying ingestion time transforms and 
+   filters are out of scope for this tutorial.
+
+8. The Configure schema settings are where you configure what [dimensions](../ingestion/index.md#dimensions) 
+   and [metrics](../ingestion/index.md#metrics) are ingested. The outcome of this configuration represents exactly how the 
+   data will appear in Druid after ingestion. 
+
+   Since our dataset is very small, you can turn off [rollup](../ingestion/index.md#rollup) 
+   by unsetting the **Rollup** switch and confirming the change when prompted.
+
+   ![Data loader schema](../assets/tutorial-batch-data-loader-05.png "Data loader schema")
+
+
+10. Click **Next: Partition** to configure how the data will be split into segments. In this case, choose `DAY` as 
+    the **Segment Granularity**. 
+
+    ![Data loader partition](../assets/tutorial-batch-data-loader-06.png "Data loader partition")
+
+    Since this is a small dataset, we can have just a single segment, which is what selecting `DAY` as the 
+    segment granularity gives us. 
+
+11. Click **Next: Tune** and **Next: Publish**.
+
+12. The Publish settings are where you can specify the datasource name in Druid. Change the default from `wikiticker-2015-09-12-sampled` 
```

Comment:
Perhaps incorporate the original sentence "Let's name this datasource wikipedia" here with "and change the default from `wikiticker-2015-09-12-sampled`" ?  The original sentence makes it more clear that the user has a choice here.

Reply:
Done. Alternatively, I wondered if we should call it something unique, like wikipedia-batchfile, or something, to allow the tutorial datasources to live together without name collisions. Perhaps for later...